src/consensus/params.h

```diff
@@ -39,6 +39,7 @@ struct Params {
     uint256 hashGenesisBlock;
     int nRewardsStart;
     int nDynodePaymentsStartBlock;
+    int nMinCountDynodesPaymentStart;
```

Comment:
Is nMinCountDynodesPaymentStart being enforced?

Reply:
No. Needs work. I just stubbed the int and set the param. Placed a check in dynode-payments.cpp.